url.bs

```diff
@@ -525,11 +525,9 @@ decisions made, i.e. whether to use the <a>same site</a> or <a>schemelessly same
 <h3 id=idna>IDNA</h3>
 
 <p>The <dfn id=concept-domain-to-ascii>domain to ASCII</dfn> algorithm, given a <a>string</a>
-<var>domain</var> and optionally a boolean <var>beStrict</var>, runs these steps:
+<var>domain</var> and an optional boolean <var>beStrict</var> (default false), runs these steps:
```

Comment:
Needs dfn (and since this is internal you can also update the call sites)

Reply:
Infra doesn't require dfn if they are positional.

Reply:
Sorry, that totally slipped my mind.